project/plugins.sbt

```diff
@@ -13,18 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
+resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-resolvers += Resolver.url("HMRC Sbt Plugin Releases", url("https://dl.bintray.com/hmrc/sbt-plugin-releases"))(Resolver.ivyStylePatterns)
 
-resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.16.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.19.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "0.19.0")
-
-addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.5.1")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.1")
```

Comment:
WARNING
Could not upgrade to latest sbt-scoverage version because this project is still built using the old sbt 0.13